modules/odc-customizing-available-cluster-roles-using-developer-perspective.adoc

```diff
@@ -0,0 +1,34 @@
+// Module included in the following assemblies:
+//
+// applications/projects/working-with-projects.adoc
+
+[id="odc-customizing-available-cluster-roles-using-developer-perspective_{context}"]
+= Customizing the available cluster roles using the Developer perspective
+
+Customize the cluster role options of an existing project mentioned in the *Project* > *Project access* tab > *Role* drop down, by editing the YAML code in the `console.openshift.io/display-name: v1`.
```

Comment:
This abstract seems more like a procedure step to me. 

How about giving a one sentence overview of why they'd customize the cluster role options? Something like:

`To have more control over your projects and to make changes to your project based off your needs, customize the cluster role options of an existing project.` 

I'd then scratch this last sentence from the abstract and make it step one of the procedure. 

1. Select  *Project* > *Project access* tab > *Role* drop down and edit the  YAML code in the `console.openshift.io/display-name: v1`.
2. Search for "*Console*" in the *Search* view > *Resources* drop down.
3. So on and so forth.

Reply:
Sounds fair. Let me rephrase a bit more based on your example.